server.dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM mcr.microsoft.com/dotnet/sdk:6.0
+RUN mkdir /app
+WORKDIR /app
+COPY . .
+RUN curl -sL https://deb.nodesource.com/setup_18.x | bash -
+RUN apt-get install -y nodejs
+RUN apt update && apt install -y wget bash libatomic1 procps
+RUN wget -q "https://cdn.altv.mp/server/release/x64_linux/altv-server"
```

Comment:
Pretty sure you should be able to do `npm run update` here?

Reply:
this is added by next commit 👍